JSTests/stress/function-constructor-name.js

```diff
@@ -31,6 +31,6 @@ return 42
 var agf = AsyncGeneratorFunction(`return 42`);
 shouldBe(typeof anonymous, `undefined`);
 shouldBe(agf.toString(),
-`async function* anonymous() {
+`async function*anonymous() {
```

Comment:
Those two cases used to work before `function* `, and `async function* ` where prefixing the rest of the function definition (from name or parameter list). So it didn't matter how the function was written (i.e. `function*` or `function *`, etc.).

Now we're actually starting from the actual "start" of the function definition, it matters. And function constructor are generated internally by the runtime, in FunctionConstructor.cpp::stringifyFunction. For a reason I don't know (probably arbitrary) function generators are `function *` and async function generator are `async function*`. That's why I had to update the spaces between the times symbol.  

If preferred, I can revert this change and update FunctionConstructor instead. I don't know what can be the consequences elsewhere/on other debugging tools though.

I understand toString behavior of function constructor is not specified anyway (V8 for instance generate totally different output:

```
> let fga =async function*(){ console.log('foo')}
> fga.constructor.toString()
'function AsyncGeneratorFunction() { [native code] }'
```